README.md

```diff
@@ -40,6 +40,32 @@ This will start a webserver on [http://127.0.0.1:4000/](http://127.0.0.1:4000/).
 To view the website open [http://127.0.0.1:4000/](http://127.0.0.1:4000/) in your favorite web browser.
 This will also automatically rebuild the project when a change to the source is detected.
 
+If port 4000 is not available on your computer use the ``--port`` option. For example:
+
+```bash
+./run --port 4020
+```
+
+You may also find adding the ``--livereload`` option useful as an alternative
```

Comment:
Doesn't it default to livereload? Otherwise, I'm wondering if we should just add it... not something to worry about for this PR though.